billing_inc.php

```diff
@@ -218,7 +227,7 @@
 	</tr>
 <?php
 		foreach ( $t_bugnote_stats['issues'] as $t_issue_id => $t_issue ) {
-			$t_project_info = ( !isset( $f_bug_id ) && $f_project_id == ALL_PROJECTS ) ? '[' . project_get_name( $t_issue['project_id'] ) . ']' . lang_get( 'word_separator' ) : '';
+			$t_project_info = ( (!isset( $f_bug_id ) && $f_project_id == ALL_PROJECTS) || $f_subprojects ) ? '[' . project_get_name( $t_issue['project_id'] ) . ']' . lang_get( 'word_separator' ) : '';
```

Comment:
I don't think this expression is correct.  I would expect:

    ( !isset( $f_bug_id ) && ( ( $f_project_id == ALL_PROJECTS ) || $f_subprojects )